packages/@uppy/companion/src/uppy.js

```diff
@@ -88,7 +88,7 @@ module.exports.app = (options = {}) => {
   app.get('/:providerName/logout', middlewares.hasSessionAndProvider, middlewares.gentleVerifyToken, controllers.logout)
   app.get('/:providerName/authorized', middlewares.hasSessionAndProvider, middlewares.gentleVerifyToken, controllers.authorized)
   app.get('/:providerName/list/:id?', middlewares.hasSessionAndProvider, middlewares.verifyToken, controllers.list)
-  app.post('/:providerName/get/:id', middlewares.hasSessionAndProvider, middlewares.verifyToken, controllers.get)
+  app.post('/:providerName/get/:id', middlewares.hasSessionAndProvider, middlewares.gentleVerifyToken, controllers.get)
```

Comment:
do you mind sharing what is the reason for this change?

Reply:
oh crap 💩, that was sloppy of me. 

I was trying to solve #1303 and commited that by accident. I'll remove it now :)